.github/workflows/msys2.yml

```diff
@@ -0,0 +1,57 @@
+name: msys2
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
At only 12 mins of time we should make this a callable workflow from the cmake.yml.